annotate/__main__.ily

```diff
@@ -113,16 +107,28 @@ annotate =
             (else
              ;; item is symbol list: annotate the next item of the given grob name
              #{ \once \override #item #'input-annotation = #props #}))))
-         ;; If set, add automatic footnote
          #{
-           #tweak-command
-           #(if (assq-ref props 'footnote-case)
-                (ann-footnote item props))
+          #tweak-command
+          #(if (assq-ref props 'footnote-offset)
+          ;; If offset present, add automatic footnote
+               (begin
+                 (if (not (assq-ref props 'footnote-text))
+                     (set! props (assoc-set! props 'footnote-text (assq-ref props 'message))))
+                 (let ((offset (assq-ref props 'footnote-offset))
+                       (text (assq-ref props 'footnote-text)))
+                 #{ \footnote #offset #text #item #})))
+          #(if (assq-ref props 'apply)
```

Comment:
It is not enough to check *if* the `'apply` property is set. If it *is* set but with a value that can't be looked up (e.g. because of a typo) LilyPond will fail in an inacceptable manner.

I think the following `getChildOption` should be used `WithFallback` (`#f`) and produce a nice `oll:warn` message, ingore the `apply` but compile successfully.